### ISSUE TITLE
Feature Request: Add Arrow Flight SQL Driver 

### ISSUE THREAD

**Issue by YuriyGavrilov** [pinterest/querybook#1359]
There is new modern SQL way to retrieve data in a faster way through Arrow Flight SQL driver. 

https://arrow.apache.org/docs/java/flight_sql_jdbc_driver.html#
https://arrow.apache.org/blog/2022/02/16/introducing-arrow-flight-sql/

**Comment by web3creator** [comment]
@YuriyGavrilov https://github.com/influxdata/flightsql-dbapi

**Comment by YuriyGavrilov** [comment]
@web3creator yep thx!, there also Dremio support it, but not sure that with same driver as querybook has.

**Comment by web3creator** [comment]
@YuriyGavrilov I just developed a version and I tested that it can be supported， need to change some code adaptation. It can be supported. You need to do some development.

**Comment by web3creator** [comment]
@YuriyGavrilov  Dremio hava bug，dremio cannot
